bot/exts/moderation/infraction/management.py

```diff
@@ -232,6 +236,9 @@ async def infraction_edit(
             # If the infraction was not marked as permanent, schedule a new expiration task
             if request_data["expires_at"]:
                 self.infractions_cog.schedule_expiration(new_infraction)
+                # Timeouts are handled by Discord itself, so we need to edit the expiry in Discord as well
+                if user and infraction["type"] == "timeout":
+                    await user.edit(reason=reason, timed_out_until=expiry)
```

Comment:
If the user is not present in the server, the infraction duration is updated in the database and a confirmation message is sent- but the actual timeout duration is not changed Discord-side (basically the same undesirable behaviour we have for editing timeout duration right now) .


We should either:
a) notify that the user has left the server
b) apply the new timeout duration when the user rejoins the server (might have to bring back some of  logic we had for `on_member_join` event from back when we used `@muted` role for muting users)

Reply:
I'm assuming it's not possible to change timeout duration for users while they are not present in the server.

Reply:
7d9bde13815dac0b8b3a71ebb90597ab06e03b25